projects/ad7134_fmc/zed/system_bd.tcl

```diff
@@ -10,7 +10,7 @@ sysid_gen_sys_init_file $sys_cstring
 
 # specify ADC resolution -- the design supports 16/24/32 bit resolutions
 
-set adc_resolution 24
+set adc_resolution 32
```

Comment:
What is going to happen, if the user change the resolution to 24? In the comment, we state that the design supports resolutions fo 16/24/32, which we still wanna do.

Reply:
What I tried to say is that we should not modify the "adc_resolution" in ststem_bd.tcl. The variable "adc_resolution" should reflect the true resolution of the device. Instead, we should round up the DATA_WIDTH of the framework to 32 bits in case of a resolution of 16 bits or 24 bits, and to 64 bits in case of 32 bits resolution. And this should be done in the common block design script.